src/containers/List/ListFilters/ListFilters.tsx

```diff
@@ -271,9 +271,9 @@ const ListFilters: React.FC<{
             return null
         }
       })}
-      {totalCount ? (
+      {totalCount !== 'loading' && (
```

Comment:
This does mean that total count jumps into view when list is first loaded, what do you think @CarlosNZ ?